docs/setup/install.md

```diff
@@ -0,0 +1,43 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Cartography Installation](#cartography-installation)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Cartography Installation
+
+Time to set up the server that will run Cartography.  Cartography _should_ work on both Linux and Windows servers, but bear in mind we've only tested it in Linux so far.  Cartography requires Python 3.6 or greater.
+
+1. **Get and install the Neo4j graph database** on your server.
+	1. Neo4j requires a JVM (JDK/JRE 11 or higher) to be installed. One option is to install [Amazon Coretto 11](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/what-is-corretto-11.html).
+
+	1. Go to the [Neo4j download page](https://neo4j.com/download-center/#releases), click "Community Server" and download Neo4j Community Edition 3.5.\*.
+
+			⚠️ At this time we run our automated tests on Neo4j version 3.5.\*.  Other versions may work but are not explicitly supported. ⚠️
+
+	1. [Install](https://neo4j.com/docs/operations-manual/current/installation/) Neo4j on the server you will run Cartography on.
+
+
+1. [Configure your data sources](config/README.md).
+
+1. **Get and run Cartography**
+
+	1. Run `pip install cartography` to install our code.
+
+	1. Finally, to sync your data:
+
+		- If you have one AWS account, run
+
+			```
+			cartography --neo4j-uri <uri for your neo4j instance; usually bolt://localhost:7687>
+			```
+
+		- If you have more than one AWS account, run
+
+			```
+			AWS_CONFIG_FILE=/path/to/your/aws/config cartography --neo4j-uri <uri for your neo4j instance; usually bolt://localhost:7687> --aws-sync-all-profiles
```

Comment:
More docs on all of the CLI options?

Reply:
Good idea.  I think I'll actually just include a mention to run `cartography --help` because each CLI param is already documented in cli.py and I want to minimize contributors needing to add the same doc to multiple places to keep this doc alive.